rush-plugins/rush-init-project-plugin/src/logic/template.schema.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Template config File",
+  "description": "The main configuration file Template management.",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+    "templates": {
+      "description": "A list of templates managed by this tool.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "displayName": {
```

Comment:
If displayName is moved to `template.json`, we'd better remote it in `init.config`. That follows our further direction.